app/src/main/res/values/strings.xml

```diff
@@ -528,10 +528,10 @@
     <string name="enable_disposed_exceptions_summary">Force reporting of undeliverable Rx exceptions outside of fragment or activity lifecycle after disposal</string>
     <string name="show_original_time_ago_title">Show original time ago on items</string>
     <string name="show_original_time_ago_summary">Original texts from services will be visible in stream items</string>
-    <string name="disable_media_tunneling_title">Disable media tunneling</string>
-    <string name="disable_media_tunneling_summary">Disable media tunneling if you experience a black screen or stuttering on video playback</string>
+    <string name="disable_media_tunneling_title">No media tunneling</string>
+    <string name="disable_media_tunneling_summary">Turn off media tunneling if you experience a black screen or stuttering video playback</string>
     <string name="show_image_indicators_title">Show image indicators</string>
-    <string name="show_image_indicators_summary">Show Picasso colored ribbons on top of images indicating their source: red for network, blue for disk and green for memory</string>
+    <string name="show_image_indicators_summary">Show Picasso-colored ribbons on top of images indicating their source: red for network, blue for disk and green for memory</string>
```

Comment:
And here too, the dash is not needed

Reply:
Picasso didn't colour ribbons, and NewPipe isn't proving it.

Reply:
I don't know in this specific instance, but generally, I think @comradekingu is right; this kind of adjectives usually has a hyphen between the words.